mock_drivers/cpp_mock_drivers/include/cpp_mock_drivers/MockDriverTemplate_h.txt

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Make sure readme describes this file. Also move into some kind of examples folder.